README.rst

```diff
@@ -60,15 +60,15 @@ Requirements
 
  * `pass <http://www.passwordstore.org/>`_
  * `xclip <http://sourceforge.net/projects/xclip>`_
- * `python <https://www.python.org/>`_
+ * `python 3 <https://www.python.org/>`_
 
 
 Installation
 ------------
 
 Installation is quite simple:
 
-    $ pip install totp-cli
+    $ pip3 install totp
```

Comment:
This alias doesn't work/exist on my system. I don't think they're a standard thing.